### ISSUE TITLE
TODO: Fix  paragraph wording/phrasing

### ISSUE THREAD

**Issue by jgarte** [hakancelikdev/unimport#221]
> this control is tox you can also provide with or let's say you are refactoring a project, unimport, black, and isort will make your job easier.

https://github.com/hakancelikdev/unimport/blob/d3de2aa32852b5794a8eac28af5787bc886f58de/docs/faq.md#long-answer

**Comment by hakancelikdev** [comment]
@jgarte do you have any suggestion ?

**Comment by jgarte** [comment]
What was trying to be said in that sentence about `tox`? I couldn't make it out. Could you rephrase it again to help explain it to me? 

I can help fix it by sending a PR once I understand the tox comment.

**Comment by hakancelikdev** [comment]
Hello, thanks for reporting this issue, I have updated the document, can you check it?
